internal/gengapic/custom_operation.go

```diff
@@ -66,12 +68,42 @@ func (g *generator) customOpPointerType() (string, error) {
 // customOpInit builds a string containing the Go code for initializing the
 // operation wrapper type with the Go identifier for a variable that is the
 // proto-defined operation type.
-func (g *generator) customOpInit(h, p string) string {
+func (g *generator) customOpInit(p, r, o string, req *descriptor.DescriptorProto, s *descriptor.ServiceDescriptorProto) {
+	h := handleName(s.GetName(), g.opts.pkgName)
 	opName := g.aux.customOp.message.GetName()
+	pt := func(f string, v ...interface{}) {
```

Comment:
seems like other spots in the file we just use p. Also `pt := g.printf` if you want an alias.

Reply:
Yes, good call, I had something else before I started using the printer and didn't clean this up properly. Thanks.